src/transformers/tokenization_utils.py

```diff
@@ -161,6 +181,46 @@ def truncate_and_pad(
         tokenizer.no_padding()
 
 
+def convert_to_tensors(batch_outputs: MutableMapping, return_tensors: Union[str, TensorType]) -> MutableMapping:
+    # Convert to TensorType
+    if not isinstance(return_tensors, TensorType):
+        return_tensors = TensorType(return_tensors)
+
+    # Get a function reference for the correct framework
+    if return_tensors == TensorType.TENSORFLOW and is_tf_available():
+        as_tensor = tf.constant
+    elif return_tensors == TensorType.PYTORCH and is_torch_available():
+        as_tensor = torch.tensor
+    elif return_tensors == TensorType.NUMPY:
+        as_tensor = np.ndarray
+    else:
+        raise ImportError(
+            "Unable to convert output to tensors format {}, PyTorch or TensorFlow is not available.".format(
+                return_tensors
+            )
+        )
+
+    # Do the tensor conversion in batch
+    for key, value in batch_outputs.items():
+        try:
+            tensor = as_tensor(value)
+
+            # at-least2d
+            if tensor.ndim > 2:
+                tensor.squeeze_(0)
+            elif tensor.ndim < 2:
+                tensor = tensor[None, :]
+
+            batch_outputs[key] = tensor
+        except ValueError:
+            if None in [item for sequence in value for item in sequence]:
```

Comment:
how do we know that `None` is padding issue?

Reply:
I suspect it comes from the fact that if padding is not set, then the actual `padding`/`padding_idx` are `None`. 

This snippet didn't change with current master, so I didn't take too much time investigating here tbh. It may make sense to dig further if we actually get None in padded input (@LysandreJik any input on this?)